src/runtime/action/index.ts

```diff
@@ -17,8 +17,8 @@
  *
  * Docs: https://svelte.dev/docs#template-syntax-element-directives-use-action
  */
-export interface ActionReturn<Parameter = any> {
-	update?: (parameter: Parameter) => void;
+interface ActionReturn<Parameter = void> {
```

Comment:
```suggestion
export interface ActionReturn<Parameter = any>
```

not sure why this change slipped in (might have been an oversight by me), I think it's better to revert it to the previous version.